[2025-05-30T01:23:06.681+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-30T01:23:06.735+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flag_check_mysql_dag.check_flag scheduled__2025-05-29T00:34:00+00:00 [queued]>
[2025-05-30T01:23:06.763+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flag_check_mysql_dag.check_flag scheduled__2025-05-29T00:34:00+00:00 [queued]>
[2025-05-30T01:23:06.765+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-05-30T01:23:06.900+0000] {taskinstance.py:2888} INFO - Executing <Task(SqlSensor): check_flag> on 2025-05-29 00:34:00+00:00
[2025-05-30T01:23:06.951+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=969) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-30T01:23:06.955+0000] {standard_task_runner.py:72} INFO - Started process 979 to run task
[2025-05-30T01:23:06.952+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flag_check_mysql_dag', 'check_flag', 'scheduled__2025-05-29T00:34:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/Sesion3/Ejemplo7.py', '--cfg-path', '/tmp/tmpqfmr_d_q']
[2025-05-30T01:23:06.964+0000] {standard_task_runner.py:105} INFO - Job 82: Subtask check_flag
[2025-05-30T01:23:07.004+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:07.104+0000] {task_command.py:467} INFO - Running <TaskInstance: flag_check_mysql_dag.check_flag scheduled__2025-05-29T00:34:00+00:00 [running]> on host 31d54648d280
[2025-05-30T01:23:07.111+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T01:23:07.678+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flag_check_mysql_dag' AIRFLOW_CTX_TASK_ID='check_flag' AIRFLOW_CTX_EXECUTION_DATE='2025-05-29T00:34:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-29T00:34:00+00:00'
[2025-05-30T01:23:07.680+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-30T01:23:07.795+0000] {base.py:84} INFO - Retrieving connection 'my_db_connection_mysql'
[2025-05-30T01:23:07.802+0000] {sql.py:96} INFO - Poking: SELECT flag FROM my_table WHERE id = 1 (with parameters None)
[2025-05-30T01:23:07.839+0000] {base.py:84} INFO - Retrieving connection 'my_db_connection_mysql'
[2025-05-30T01:23:07.874+0000] {sql.py:509} INFO - Running statement: SELECT flag FROM my_table WHERE id = 1, parameters: None
[2025-05-30T01:23:07.875+0000] {sql.py:518} INFO - Rows affected: 1
[2025-05-30T01:23:07.877+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-05-30T01:23:07.939+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-30T01:23:07.939+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flag_check_mysql_dag, task_id=check_flag, run_id=scheduled__2025-05-29T00:34:00+00:00, execution_date=20250529T003400, start_date=20250530T012306, end_date=20250530T012307
[2025-05-30T01:23:08.050+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-30T01:23:08.153+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-30T01:23:08.155+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
